.github/workflows/lint.yaml

```diff
@@ -23,8 +23,8 @@ jobs:
           apt-get -y install pkg-config libtool autoconf \
             automake g++ bison flex make bash-completion dh-autoreconf \
             debhelper devscripts wget python3-pip
-      - name: Install Python lint tools
-        run: python3 -m pip install --no-input cpplint flake8
+      #- name: Install Python lint tools
+      #  run: python3 -m pip install --no-input cpplint flake8
```

Comment:
Should be able to remove this for all unrelated tasks now (except for of course flake8 in flake8 and cpplint in cpplint).

Reply:
Although I suppose you do need them when building or you will have to reconfigure.

Reply:
Yeah exactly that hence why it was failing just now when the build for lint passed. I only removed them to confirm configure succeeded with them not present and didn't crash.